src/common/main.lua

```diff
@@ -597,12 +266,48 @@ function ant.init()
 
 		ao.send({
 			Target = msg.From,
+			-- This is out of our pattern, should be <action>-Notice
 			Action = "Reassign-Name-Submit-Notice",
 			Initiator = msg.From,
 			Name = name,
 			["Process-Id"] = antProcessIdToReassign,
 		})
 	end)
+
+	Handlers.prepend(
```

Comment:
Actually we could get rid of this

Reply:
so instead of 
```
SourceCodeTxId = SourceCodeTxId or "__INSERT_SOURCE_CODE_ID__"
```
we do
```
SourceCodeTxId = "__INSERT_SOURCE_CODE_ID__"
```